packages/components/src/components/toast/toast.tsx

```diff
@@ -3,7 +3,24 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { CSSTransition } from 'react-transition-group';
 
-const Toast = ({ children, className, is_open = true, onClose, onClick, type = 'info', timeout = 0 }) => {
+type TToast = {
+    className: string;
+    is_open: boolean;
+    onClick: (event: React.MouseEvent<HTMLElement>) => void;
+    onClose: () => void;
+    type: 'error' | 'info';
+    timeout: number;
+};
+
+const Toast = ({
+    children,
+    className,
+    is_open = true,
+    onClose,
+    onClick,
+    type = 'info',
+    timeout = 0,
+}: React.PropsWithChildren<Partial<TToast>>) => {
```

Comment:
Same as here.

Reply:
Since the entire list of props is optional, I chose wrapping the type with Partial interface, rather than adding `?`